quartz/plugins/transformers/citations.ts

```diff
@@ -0,0 +1,33 @@
+import rehypeCitation from "rehype-citation"
+import { QuartzTransformerPlugin } from "../types"
+
+export interface Options {
+  bibliographyFile: string
+  suppressBibliography: boolean
+}
+
+const defaultOptions: Options = {
```

Comment:
can we expose `options.csl` so people can choose their citation format?

Reply:
https://github.com/timlrx/rehype-citation?tab=readme-ov-file#optionscsl

Keep in mind that `options.csl` can be either a string or url. We might want to check if Quartz handles url sources properly when applying `noPopovers`, although this might be out of scope for this pull request or might be already handled by the `rehype-citation` library.

https://github.com/citation-style-language/styles

Reply:
So, I just added the `csl` option as a simple string as if I understand correctly, the rest is handled by `rehype-citation`.